source/triggers/clauses/clause_IsAtPosition.js

```diff
@@ -48,6 +50,8 @@ this.onGenerated = function( params, generator, payload ) {
         return false;
     }
 
+    this.useAxisOffset = !!params[ 2 ];
```

Comment:
It forces it to be a Boolean.  It's really not necessary in this case, but I felt like it added clarity (and prevents anybody from abusing the value, I suppose).